src/com/facebook/buck/apple/CodeSignStep.java

```diff
@@ -61,7 +62,8 @@ public CodeSignStep(
       Tool codesign,
       Optional<Tool> codesignAllocatePath,
       Optional<Path> dryRunResultsPath,
-      ImmutableList<String> codesignFlags) {
+      ImmutableList<String> codesignFlags,
+      long codesignTimeout) {
```

Comment:
instead of passing `long` which does not include the units, please use `Duration` instead.